### ISSUE TITLE
Setting Password Strength

### ISSUE THREAD

**Issue by Zero2Cool** [YAFNET/YAFNET#657]
How can we set the password strength to being X characters long and not requiring symbols, numbers or uppercase? Normally this was done in the web.config, right?

**Comment by w8tcha** [comment]
Its now in the Host Settings under  **Password Strength Settings**

**Comment by Zero2Cool** [comment]
> Its now in the Host Settings under **Password Strength Settings**

lol and I get object reference to null when I try to go into Host Settings. I'm only using the source code with a copy of my database. Crazy how complicated upgrading is for me. 🤦‍♂️

**Comment by w8tcha** [comment]
Can you post the full error? I try to reproduce the issue.

**Comment by Zero2Cool** [comment]
> Can you post the full error? I try to reproduce the issue.

I gave up and deleted the source code. Sorry, just too much of a hassle. I tried just creating a new forum with new database and even that didn't work so I must have something wrong with my computer and just don't have the time to keep spending on it.  Chalk this one up as user error.

**Comment by w8tcha** [comment]
It's odd, did you download the master or stable branch? The stable branch should be the safer option.

**Comment by Zero2Cool** [comment]
> It's odd, did you download the master or stable branch? The stable branch should be the safer option.

Yeah, I don't get it either, but no need to be wasting your time when it's clearly something on my end.  I downloaded the release version source code for YAF and Sample Application.  I tried getting YAF to build and install on new database and everything on it's own before putting it into Sample Application. But when it wouldn't go, I figured it just has to be something on my computer that's screwing it up.
